packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -243,10 +250,13 @@ async function fetchLatestSnapshotCore(
 	snapshotDownloader: (
 		finalOdspResolvedUrl: IOdspResolvedUrl,
 		storageToken: string,
+		groupIds: string[],
 		snapshotOptions: ISnapshotOptions | undefined,
 		controller?: AbortController,
 	) => Promise<ISnapshotRequestAndResponseOptions>,
 	putInCache: (valueWithEpoch: IVersionedValueWithEpoch) => Promise<void>,
+	groupIds: string[],
+	fetchSnapshotForInitialLoad?: boolean,
```

Comment:
We can give different names to "TreesLatest" and "Obtainsnapshot" event in case it was not for initial load. That would help in analysis easier probably otherwise we would need to use this Boolean along with event name in order to do analysis on boot times etc. @vladsud

Reply:
Yes, I think it would be nice. If possible, I'd actually do the following:
1. For default loading, put some boolean on event to indicate client is asking for "optimized" snapshot, i.e. it's not legacy client who depends on full snapshot.
2. For loading by groupID, issue another event.